ui/components/multichain/address-copy-button/address-copy-button.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ICON_NAMES, ButtonBase } from '../../component-library';
+import {
+  BackgroundColor,
+  IconColor,
+  TextVariant,
+  TextColor,
+  Size,
+  BorderRadius,
+  AlignItems,
+  OVERFLOW_WRAP,
+} from '../../../helpers/constants/design-system';
+import { useCopyToClipboard } from '../../../hooks/useCopyToClipboard';
+import { shortenAddress } from '../../../helpers/utils/util';
+import Tooltip from '../../ui/tooltip/tooltip';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+
+export const AddressCopyButton = ({
+  address,
+  shorten = false,
+  wrap = false,
+}) => {
+  const displayAddress = shorten ? shortenAddress(address) : address;
+  const [copied, handleCopy] = useCopyToClipboard();
+  const t = useI18nContext();
+
+  return (
+    <Tooltip position="bottom" title={copied ? t('copiedExclamation') : null}>
+      <ButtonBase
+        backgroundColor={BackgroundColor.primaryMuted}
+        onClick={() => handleCopy(address)}
+        padding={0}
+        paddingRight={3}
+        paddingLeft={3}
+        size={Size.SM}
+        className="multichain-address-copy-button"
```

Comment:
Previously if `wrap` was `true` you set the height to `auto` which I think made sense? It would help it degrade gracefully we would end up with a bubble button but at least the text wouldn't be out side the button?

So instead of this(I know I've made the address ridiculously long)

<img width="247" alt="Screenshot 2023-03-22 at 6 07 41 PM" src="https://user-images.githubusercontent.com/8112138/227072908-2904bb46-363b-46f6-ac24-1b83f2f07c71.png">

It would be more like
<img width="311" alt="Screenshot 2023-03-22 at 6 08 37 PM" src="https://user-images.githubusercontent.com/8112138/227072932-d60aa45f-5fe0-4278-8f23-40b924ed702f.png">

But I guess that's a design decision? cc @SaraCheikh @Akatori-Design

Reply:
If it breaks it should gracefuly break, so yea I think we should keep the text in the button,